src/dolphin/manager.ts

```diff
@@ -151,7 +151,7 @@ export class DolphinManager {
     }
   }
 
-  public async reinstallDolphin(launchType: DolphinLaunchType) {
+  public async reinstallDolphin(launchType: DolphinLaunchType, cleanInstall: boolean) {
```

Comment:
```suggestion
  public async reinstallDolphin(launchType: DolphinLaunchType, cleanInstall?: boolean) {
```
nit: can we keep this optional?